### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by yash6102000**

### CODE REVIEW

File: app/components/results/AttendedCandidateListItem.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ const AttendedCandidateListItem = ({
     <div className="col-span-full">
       <div
         id="groupByTestRow"
-        className="col-span-full grid grid-cols-12 border-t-[1px] border-solid border-[#E5E7EB] bg-white px-12 py-4"
+        className="col-span-full grid grid-cols-12 gap-3 border-t-[1px] border-solid border-[#E5E7EB] bg-white px-12 py-4"
```
Comment (arpita-20) [inline app/components/results/AttendedCandidateListItem.tsx:20]: remove hard coded value

File: app/components/sections/add-question/AddQuestionInSection.tsx [hunk 2]
```diff
@@ -100,7 +96,9 @@ const AddQuestionInSection = () => {
         flag == 0 &&
         getQuestionType(selectedTypeOfQuestion) === QuestionTypes.multipleChoice
       ) {
-        toast.error('Select the Option', { toastId: 'correctOptionRequired' })
+        toast.error('Select the Correct Option', {
```
Comment (arpita-20) [inline app/components/sections/add-question/AddQuestionInSection.tsx:99]: add 'select the correct option' in  common constant file

File: app/components/sections/add-question/AddQuestionInSection.tsx [hunk 3]
```diff
@@ -109,7 +107,9 @@ const AddQuestionInSection = () => {
           QuestionTypes.singleChoice &&
         !singleChoiceAnswer
       ) {
-        toast.error('Select the Option', { toastId: 'correctOptionsRequired' })
+        toast.error('Select the Correct Option', {
```
Comment (arpita-20) [inline app/components/sections/add-question/AddQuestionInSection.tsx:110]: add 'select the correct option' in common constant file

File: app/components/sections/add-question/OptionForQuestion.tsx [hunk 4]
```diff
@@ -47,6 +47,9 @@ export default function OptionForQuestion({
         QuestionTypes.multipleChoice ||
       getQuestionType(selectedTypeOfQuestion) === QuestionTypes.singleChoice
     ) {
+      if (options.length > 5) {
+        return toast.error('you can add maximum six options only.', { toastId })
```
Comment (arpita-20) [inline app/components/sections/add-question/OptionForQuestion.tsx:51]: add this in common constant file

**Review by arpita-20 — CHANGES_REQUESTED** [review]

**Review by richa-copods — DISMISSED** [review]

**Review by anuragP04 — APPROVED** [review]

**Pull request merged**